_includes/formula-linux.html

```diff
@@ -0,0 +1,3 @@
+<td><a href="{{ site.baseurl }}/formula-linux/{{ include.formula.name }}">{{ include.formula.name }}</a></td>
```

Comment:
Maybe files like this could be generated from the script?

Reply:
I understand very little about how any of the `_includes` files work, I included them based on @EricFromCanada's review earlier and his refactor of all that stuff. If we do it for Linux then we should probably do it for macOS too? Right now there's `_includes/formula.html` and `_includes/formula-linux.html`. I can try and look into it, though. :+1:

Reply:
https://jekyllrb.com/docs/includes/#passing-parameters-to-includes may be what you need?

Reply:
I've dispensed with this file entirely in favour of implementing thing I learned from the above docs in 21155cb. Thanks, Mike. :books: :eyeglasses: